CHANGELOG.md

```diff
@@ -6,10 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [2.2.0] - 2022-09-19
+### Added
+- Add support for the Sepolia network ([#60](https://github.com/MetaMask/etherscan-link/pull/60))
+
 ## [2.1.0] - 2021-05-07
 ### Added
 
-- Add methods (getBlockExplorerLink, getAccountLink, getTokenTrackerLink) that select correct block explorer link format based on input params ([#31](https://github.com/MetaMask/etherscan-link/pull/44))
+- Add methods (getBlockExplorerLink, getAccountLink, getTokenTrackerLink) that select correct block explorer link format based on input params ([#44](https://github.com/MetaMask/etherscan-link/pull/44))
```

Comment:
Little link fix

Reply:
@danjm no annoyance at all! :) Updated, will make a separate PR after this is merged.